### PR TITLE
feat(with-feature): option for multiple browsers

### PR DESCRIPTION
**Pull request by matangeorgi**
An option to run multiple browsers within the withFeature, using env vars.
Chromium will run as the default browser, when no env vars are being passed.

### CODE REVIEW

File: packages/test-kit/src/with-feature.ts [hunk 1]
```diff
@@ -149,6 +149,7 @@ export function withFeature(withFeatureOptions: IWithFeatureOptions = {}) {
     const disposeAfterEach = createDisposables();
     const envDebugMode = 'DEBUG' in process.env;
     const debugMode = !!process.env.DEBUG;
+    const browserToRun = process.env.BROWSER as 'chromium' | 'firefox' | 'webkit' | undefined;
```
Comment (daomry) [inline packages/test-kit/src/with-feature.ts:152]: ts will not check it in runtime, check that the value ius a supported browser and throw a nice error message with a list of valid options

**Review by daomry — CHANGES_REQUESTED** [review]

File: packages/test-kit/src/with-feature.ts (hunk 1 above)
Comment (matangeorgi) [reply packages/test-kit/src/with-feature.ts]: Done :)

**Review by daomry — APPROVED** [review]
Well done. I moved the browser related code to its own file and made slight adjustments, let me know what you think

File: packages/test-kit/src/supported-browsers.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+export type SupportedBrowser = typeof supportedBrowsers[number];
+
+export function isValidBrowserName(browserName: string): browserName is SupportedBrowser {
+    return (supportedBrowsers as ReadonlyArray<string>).includes(browserName);
+}
+export function validateBrowser(name: string): SupportedBrowser {
+    if (isValidBrowserName(name)) {
+        return name;
+    }
+    throw new Error(`Invalid browser name was entered as env var: ${name} \n
```
Comment (thehulke) [inline packages/test-kit/src/supported-browsers.ts:11]:
```suggestion
    throw new Error(`Invalid browser name was entered as env var: ${name} ,
```

**Review by AviVahl — APPROVED** [review]
fix conflicts

**Pull request merged**